README.md

```diff
@@ -39,16 +39,38 @@ jobs:
         branch: ${{ github.ref }}
 ```
 
+An example workflow to use the force-with-lease paremater to force push to an repository:
+
+```yaml
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+	  fetch-depth: 0
```

Comment:
Can we fix the mixed indentation here with tabs/spaces

Reply:
Done